main.js

```diff
@@ -113,6 +113,15 @@ function createWindow() {
         contentView.webContents.insertCSS(commonLayout.toString())
     })
 
+    contentView.webContents.on('new-window', (event, url, frameName, disposition, options) => {
+        // Open URL in default browser when user clicks on a hyperlink
+        event.preventDefault()
+        // Don't open url for attatchments, will fail.
+        if (!url.includes("rocketchat.daronmont.com.au/file-upload/")) {
```

Comment:
Can't approve a pull request where the server URL isn't generalised. Just use `settings[0]/file-upload/` for now until we get JSON going for configs